updater.go

```diff
@@ -176,7 +186,7 @@ func sleepUpdater(approxWakeup time.Time, st *stopper.Stopper) (stopped bool) {
 
 // update fetches all the vulnerabilities from the registered fetchers, upserts
 // them into the database and then sends notifications.
-func update(datastore database.Datastore, firstUpdate bool) {
+func update(datastore database.Datastore, firstUpdate bool) (status bool) {
```

Comment:
I'm not sure you've done anything with status? Is the last line of this function supposed to be `return true`?

Reply:
@jzelinskie please kindly let me know your thought.

Reply:
Oh sorry. I literally just missed that line because it was under the GitHub fold.

If we rename status to something more descriptive like "updateSuccessful", then this LGTM